regression/cbmc-java/exceptions26/test.desc

```diff
@@ -0,0 +1,7 @@
+CORE
+test.class
+^VERIFICATION SUCCESSFUL$
+--
+^warning: ignoring
+--
+This test verifies that catches are executed in the correct order
```

Comment:
Nit pick: missing newline

Reply:
I added it, but it's not showing here (same for the other one below)

Reply:
Looks fine now, github is no longer showing that red warning sign.